Library/Formula/open-scene-graph.rb

```diff
@@ -62,7 +62,14 @@ def install
     end
 
     if build.with? "collada-dom"
-      args << "-DCOLLADA_INCLUDE_DIR=#{Formula["collada-dom"].opt_include}/collada-dom"
+      # https://github.com/Homebrew/homebrew/issues/43536
+      collada_include_candidates = Dir.glob("#{Formula["collada-dom"].opt_include}/collada-dom*")
+      if collada_include_candidates.empty?
+        odie "Could not locate collada-dom include directory"
+      end
+      collada_include_dir = collada_include_candidates[0]
```

Comment:
My impression was that using `.first` in these instances is more common than array indexing with `[0]`. You could also move the preceding check after this statement, making the code a bit more compact:

```ruby
collada_include_dir = Dir.glob("#{Formula["collada-dom"].opt_include}/collada-dom*").first
raise "Could not locate collada-dom include directory" unless collada_include_dir
```

(But maybe the code is too wide to warrant using `unless` as a statement modifier.)